### ISSUE TITLE
Mouseover mob icons - description not appearing

### ISSUE THREAD

**Issue by JoBu92** [Nnoggie/MythicDungeonTools#233]
Mousing over Mobs in MDT doesn't show the popup with the information to the mobs.

Bug introduced (for me) with v.3.4.8 - All versions before that work in regards to said issue.

Described Issue v.3.4.8 and above:
![grafik](https://user-images.githubusercontent.com/56517331/112924707-edcbe900-9110-11eb-8026-2a981518021a.png)

v,3.4.7:
![grafik](https://user-images.githubusercontent.com/56517331/112924951-51561680-9111-11eb-8eec-fb2afcdb6760.png)

BugSack descriptions:
![grafik](https://user-images.githubusercontent.com/56517331/112924504-8d3cac00-9110-11eb-92f3-4cdcf3b28d7c.png)
![grafik](https://user-images.githubusercontent.com/56517331/112924568-a8a7b700-9110-11eb-803d-3ad3001d4cd2.png)

Thank you for the help!
Cheers

**Comment by Nnoggie** [comment]
The most uptodate version of mdt is v3.5.4. Did you download MDT from a different source than curseforge?

**Comment by JoBu92** [comment]
I'm using WowUp.io - And 3.5.4 didn't work either. Forgot to say that it started from v3.4.8 up to the current one.

**Comment by Nnoggie** [comment]
Download MDT from https://www.curseforge.com/wow/addons/mythic-dungeon-tools/files

**Comment by JoBu92** [comment]
I tried that today and it didn't work unfortunately.
But I found out which addon is interfering and causing the error: It is Wind Dungeon Helper. In the options menu of the addon you have a 'MDT Helper' section with the option 'Translate Existing Data'. This option is screwing with MDT.
In case anyone else has this problem here might be a solution.

Thank you for your time Nnoggie! Keep up the good work.
